app/src/main/res/layout-sw600dp-land/add_profile_activity.xml

```diff
@@ -0,0 +1,216 @@
+            android:paddingStart="12dp"
+            android:paddingTop="12dp"
+            android:src="@drawable/rounded_white_background_with_shadow"
+            app:layout_constraintBottom_toBottomOf="@+id/add_profile_activity_user_image_view"
+            app:layout_constraintEnd_toEndOf="@+id/add_profile_activity_user_image_view"
+            app:layout_constraintRight_toRightOf="@+id/add_profile_activity_user_image_view" />
+
+          <TextView
+            android:id="@+id/add_profile_activity_required_heading_text_view"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="32dp"
+            android:text="@string/add_profile_required"
+            android:textColor="@color/oppiaPrimaryText"
+            android:textSize="12sp"
+            app:layout_constraintTop_toBottomOf="@+id/add_profile_activity_user_image_view" />
+
+          <org.oppia.android.app.profile.ProfileInputView
+            android:id="@+id/add_profile_activity_user_name_profile_input_view"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="16dp"
+            app:inputLength="36"
+            app:label="@string/add_profile_input_name"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/add_profile_activity_required_heading_text_view"
+            profile:error="@{viewModel.nameErrorMsg}"
+            profile:singleLine="@{true}" />
+
+          <CheckBox
+            android:id="@+id/add_profile_activity_pin_check_box"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="48dp"
+            android:minHeight="48dp"
+            android:text="@string/add_profile_create_a_3_digit_pin"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/add_profile_activity_user_name_profile_input_view" />
+
+          <ImageView
+            android:id="@+id/add_profile_activity_info_image_view"
+            android:layout_width="48dp"
+            android:layout_height="48dp"
+            android:padding="12dp"
+            android:src="@drawable/ic_info_icon"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="@+id/add_profile_activity_pin_check_box" />
+
+          <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/add_profile_activity_pin_constraint_layout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:visibility="@{viewModel.createPin ? View.VISIBLE : View.GONE}"
+            app:layout_constraintTop_toBottomOf="@+id/add_profile_activity_pin_check_box">
```

Comment:
Required because of first comment.
```
            app:layout_constraintEnd_toEndOf="parent"
            app:layout_constraintStart_toStartOf="parent"
            app:layout_constraintTop_toBottomOf="@+id/add_profile_activity_pin_check_box">
```

Reply:
Done